recipe/meta.yaml

```diff
@@ -12,18 +12,19 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
+  noarch: python
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy
     - pandas
```

Comment:
Are these still needed at build time?

Reply:
Probably not. Engarde is pure-python, and doesn't import NumPy during setup.py, so should it be run-only? Same with pandas?

Reply:
Yep, both can be dropped then.